tests/other/request.test.js

```diff
@@ -1367,6 +1381,148 @@ describe('Trips stats', () => {
         expect(error).to.be.a('string');
         expect(error).to.equal(invalidTripsStatsEndDate);
         done();
+  }); 
+}); 
+});
+describe('update open travel request', () => {
+  before('Insert sample request in db', () => {
+    sequelize.query(INSERT_REQUEST);
+  });
+  it('should not update open travel request when token is missing', (done) => {
+    chai
+      .request(server)
+      .patch(`/api/trips/${requestId}`)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(badRequest);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('error').to.equal(tokenMissing);
+        done();
+      });
+  });
+  it('should not update open travel request when token is invalid', (done) => {
+    chai
+      .request(server)
+      .patch(`/api/trips/${requestId}`)
+      .set('Authorization', `Bearer ${wrongToken}`)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(badRequest);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('error').to.equal(tokenInvalid);
+        done();
+      });
+  });
+  it('should update open travel request', (done) => {
+    chai
+      .request(server)
+      .patch(`/api/trips/${requestId}`)
+      .set('Authorization', authToken)
+      .send(travelUpdated)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(badRequest);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('error').to.equal(duplicateTripRequest);
+        done();
+      });
+  });
+  it('should update open travel request', (done) => {
+    chai
+      .request(server)
+      .patch(`/api/trips/${requestId2}`)
+      .set('Authorization', authToken)
+      .send(duplicateUpdate)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(ok);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('message').to.equal(requestUpdated);
+        done();
+      });
+  });
+  it('should not update approved or rejected travel request', (done) => {
+    chai
+      .request(server)
+      .patch('/api/trips/13')
```

Comment:
Hard coded id's may introduce errors in the future. Same as the one in @descholar-ceo  code.